SignalMessaging/ViewControllers/OWSViewController.m

```diff
@@ -93,7 +93,7 @@ - (void)autoPinViewToBottomOfViewControllerOrKeyboard:(UIView *)view
                                                object:nil];
 
     self.bottomLayoutView = view;
-    self.bottomLayoutConstraint = [view autoPinEdge:ALEdgeBottom toEdge:ALEdgeBottom ofView:self.view];
+    self.bottomLayoutConstraint = [view autoPinToBottomLayoutGuideOfViewController:self withInset:0.f];
```

Comment:
This looks correct on iPhoneX with and without the keyboard popped?

Reply:
Yes, sorry I meant to call this out.  This addresses an issue on iPhone X around the "bottom pinned" view (incorrectly) hiding behind "the notch".  It turns out that some "bottom pinned" views want to avoid the notch; others don't.  I've tweaked this.